### PR TITLE
LPD-37314 - Migrate SPA Poshi tests

### PR DESCRIPTION
**Pull request by juanjofgliferay**
https://liferay.atlassian.net/browse/LPD-37314

This PR only includes the migration of SPA related tests. Other tests in SPA.testcase file will be migrated to a new `frontend-taglib-sample-web` module

### CODE REVIEW

File: modules/test/playwright/tests/frontend-js-spa-web/settings.spec.ts [hunk 1]
```diff
@@ -73,3 +73,69 @@ test(
...
+	{tag: '@LPS-67072'},
+	async ({page, systemSettingsPage}) => {
+		await test.step('Navigate to SPA Settings page', async () => {
+			await systemSettingsPage.goToSystemSetting(
+				'Infrastructure',
+				'Frontend SPA Infrastructure'
+			);
+		});
+
+		const userNotificationTimeoutLabel = page.getByLabel(
```
Comment (juanjofgliferay) [inline modules/test/playwright/tests/frontend-js-spa-web/settings.spec.ts:88]: We have only a few SPA settings page locator, but we can consider creating a `SettingsPage.ts` to hold all of them if the number increase.

Comment (markocikos) [reply modules/test/playwright/tests/frontend-js-spa-web/settings.spec.ts:88]: Yep. Very optional for this PR, can be a rule of three.

**Review by markocikos — APPROVED** [review]
@juanjofgliferay LGTM! :sparkles:

**Comment by juanjofgliferay** [comment]
ci:forward

**Pull request closed without merging**